src/chain.js

```diff
@@ -75,19 +79,30 @@ function startServer(options) {
         return
       }
 
-      var data = {
-        mnemonic: state.mnemonic,
-        hdPath: state.wallet_hdpath,
-        privateKeys: {}
-      }
+      let privateKeys = {}
 
       var accounts = state.accounts;
       var addresses = Object.keys(accounts);
 
       addresses.forEach(function(address, index) {
-        data.privateKeys[address] = accounts[address].secretKey.toString("hex")
+        privateKeys[address] = accounts[address].secretKey.toString("hex")
       });
 
+      let data = Object.assign({}, server.provider.options)
```

Comment:
This change wasn't strictly necessary, but I did some work over at trufflesuite/ganache-core@7223c45a and trufflesuite/ganache-core@d0fa6dd4 to fill out the options object after the fact with the in-use configuration. This makes it so we don't need to hunt around `ganache-core` for the bits of info we need.